src/platform/intel/cavs/alternate_reset_vector.S

```diff
@@ -0,0 +1,330 @@
+/******************************************************************************
+INTEL CONFIDENTIAL
+Copyright 2020 Intel Corporation All Rights Reserved.
+
+The source code contained or described herein and all documents related to the
+source code ("Material") are owned by Intel Corporation or its suppliers or
+licensors. Title to the Material remains with Intel Corporation or its
+suppliers and licensors. The Material may contain trade secrets and
+proprietary and confidential information of Intel Corporation and its
+suppliers and licensors, and is protected by worldwide copyright and trade
+secret laws and treaty provisions. No part of the Material may be used, copied,
+reproduced, modified, published, uploaded, posted, transmitted, distributed,
+or disclosed in any way without Intel's prior express written permission.
+
+No license under any patent, copyright, trade secret or other intellectual
+property right is granted to or conferred upon you by disclosure or delivery of
+the Materials, either expressly, by implication, inducement, estoppel or
+otherwise. Any license under such intellectual property rights must be express
+and approved by Intel in writing
+
+*******************************************************************************/
+
+#include <config.h>
+#include <sof/bit.h>
+#include <sof/common.h>
+#include <sof/drivers/interrupt.h>
+#if CONFIG_BOOT_LOADER && !CONFIG_VM_ROM
+#include <sof/lib/memory.h>
+#include <sof/lib/shim.h>
+#endif
+#include <xtensa/cacheattrasm.h>
+#include <xtensa/cacheasm.h>
+#include <xtensa/coreasm.h>
+#include <xtensa/config/system.h>
+#include <xtensa/xdm-regs.h>
+#include <xtensa/xtensa-xer.h>
+#include <xtensa/xtruntime-core-state.h>
+
+#include "xtos-internal.h"
+
+#define DSP_MS_IDC_TAR_FROM_MASTER_OFFSET 0x00
+
+.type   slave_core_init, @function
+.type   g_shared_vecbase_ptr, @object
+
+.section        .AlternateResetVector.text, "ax"
+
+    /* Alignemt and padding for 4 dwords is in order to meet
+     * lpsram header requiremetns  */
+_LpsramHeader:
+     j    _AltResetVector
```

Comment:
Indent is 4 spaces here, but should be 1 tab (rendered as 8 spaces) as in other .S files

Reply:
fixed